python/GafferSceneUITest/CropWindowToolTest.py

```diff
@@ -152,9 +155,16 @@ def testImageViewStatus( self ) :
 
 		# Valid options path
 
-		self.waitForIdle( 1000 )
-		self.assertEqual( tool.status(), "Info: Editing <b>options.options.renderCropWindow.value</b>" )
+		with IECore.CapturingMessageHandler() as mh :
```

Comment:
Looks like we still have the test failure on CI - I think we're going to have to broaden the scope of this `with` block, potentially all the way up to where we make the window.